File: CoreJava/employee-polymorphic-app/src/com/techlab/employee/Analyast.java
package com.techlab.employee;

public class Analyast extends Employee  {
	private final float PERKS = 20; 
	public Analyast(int empId, String empName) {
		super(empId, empName);
	}
	
	public float getPERKS() {
		return PERKS;
	}

	@Override
	public void calculateSalary() {
		grossSalary = BASICPAY+ (BASICPAY*(PERKS/100));
	}

	
}

File: C#Basics/Account-Up-DownCasting-App/Account-Up-DownCasting-App/Analyast.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Account_Up_DownCasting_App
{
    class Analyast : Employee
    {
		private const float PERKS = 20;

		public Analyast(int empId, String empName) : base(empId, empName) { }
		

		public float getPERKS()
		{
			return PERKS;
		}

		
		public override void calculateSalary()
		{
			grossSalary = BASICPAY + (BASICPAY * (PERKS / 100));
		}

    }
}
